### PR TITLE
[generate] update docstring of `SequenceBiasLogitsProcessor`

### LINKED ISSUE #35432: apply class transformers.SequenceBiasLogitsProcessor on Qwen model
### Feature request

I notice that your wrote:
In order to get the token ids of the sequences that you want to bias, make sure to set add_prefix_space=True when initializing the tokenizer, and use tokenizer(bad_words, add_special_tokens=False).input_ids. The add_prefix_space argument is only supported for some slow tokenizers, as fast tokenizers’ prefixing behaviours come from pre tokenizers. Read more [here](https://huggingface.co/docs/tokenizers/api/pre-tokenizers).

So if I am using Qwen model, since its tokenizer is based on fast tokenizers, I can't use this bias logits feature?

Is there anyway I can use the Qwen model and still use this feature?

### Motivation

the old code is not working on the new model like qwen, want update.

### Your contribution

none

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

Fixes #35432 

The docstring of `SequenceBiasLogitsProcessor` was outdated -- the example didn't work with fast tokenizers, and fast tokenizers are now the norm. This PR updates the example and tips, especially to be used with fast tokenizers.

### CODE REVIEW

File: src/transformers/generation/logits_process.py [hunk 1]
```diff
@@ -1076,37 +1075,40 @@ class SequenceBiasLogitsProcessor(LogitsProcessor):
     ```python
     >>> from transformers import AutoTokenizer, AutoModelForCausalLM
 
-    >>> model = AutoModelForCausalLM.from_pretrained("openai-community/gpt2")
-    >>> tokenizer = AutoTokenizer.from_pretrained("openai-community/gpt2")
+    >>> model = AutoModelForCausalLM.from_pretrained("Qwen/Qwen2.5-0.5B-Instruct")
+    >>> tokenizer = AutoTokenizer.from_pretrained("Qwen/Qwen2.5-0.5B-Instruct")
```
Comment (gante) [inline src/transformers/generation/logits_process.py:1079]: this is a fast tokenizer

**Review by ArthurZucker — APPROVED** [review]
